kernel/posix/pthread_sched.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <pthread.h>
+
+/*
+ *  ======== sched_get_priority_min ========
+ *  Provide Minimum Priority for the given policy
+ *
+ *  In case of Cooperative priority value returned
+ *  is negative of the priority value.
+ *
+ *  returns:	priority corresponing to policy
+ *		-EINVAL for Error
+ */
+int sched_get_priority_min(int policy)
+{
+
+	int priority = -EINVAL;
+
+	if (!((policy == CONFIG_COOP_ENABLED) || (policy == SCHED_PREMPT))) {
```

Comment:
CONFIG_COOP_ENABLED sounds like a Zephyr thing, rather than a POSIX thing.  Is it correct here?